src/server/game/Accounts/AccountMgr.h

```diff
@@ -21,6 +21,7 @@ enum AccountOpResult
 };
 
 #define MAX_ACCOUNT_STR 20
+#define MAX_PASS_STR 16
```

Comment:
How its related to the PR?

Reply:
No it was my bad, i've created the PR with my fork on master branch, now this PR is merged than this snippet will not be considered.